charts/chronograf/README.md

```diff
@@ -1,58 +1,52 @@
+The Chronograf Helm chart uses the [Helm](https://helm.sh) package manager to bootstrap a Chronograf deployment and service on a [Kubernetes](http://kubernetes.io) cluster.
 
 ## Prerequisites
 
 - Kubernetes 1.4+
-- PV provisioner support in the underlying infrastructure (optional)
+- (Optional) PersistentVolume (PV) provisioner support in the underlying infrastructure
 
-## Installing the Chart
+## Install the chart
 
-To install the chart with the release name `my-release`:
+1. Add the InfluxData Helm repository:
 
-```bash
-helm upgrade --install my-release influxdata/chronograf
-```
+   ```bash
+   helm repo add influxdata https://helm.influxdata.com/
+   ```
+
+2. Run the following command, providing a name for your Chronograf release:
 
-The command deploys Chronograf on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+   ```bash
+   helm upgrade --install my-release influxdata/chronograf
+   ```
+    
+   > **Tip**: `--install` can be shortened to `-i`.
 
-> **Tip**: List all releases using `helm list`
+   This command deploys Chronograf on the Kubernetes cluster using the default configuration. To find parameters you can configure during installation, see [Configure the chart](#configure-the-chart).
 
-## Uninstalling the Chart
+  > **Tip**: To view all Helm chart releases, run `helm list`.
 
-To uninstall/delete the `my-release` deployment:
+## Uninstall the chart
+
+To uninstall the `my-release` deployment, use the following command:
 
 ```bash
 helm uninstall my-release
 ```
 
-The command removes all the Kubernetes components associated with the chart and deletes the release.
-
-## Configuration
+This command removes all Kubernetes components associated with the chart and deletes the release.
 
-The configurable parameters of the Chronograf chart and
-their descriptions can be seen in `values.yaml`. The [full image documentation](https://quay.io/influxdb/chronograf) contains more information about running Chronograf in docker.
+## Configure the chart
 
-The following table lists the configurable parameters of the chronograf chart and their default values.
+The following table lists configurable parameters, their descriptions, and their default values stored in `values.yaml`.
 
 | Parameter                    | Description                                                                                               | Default                                     |
 |:-----------------------------|:----------------------------------------------------------------------------------------------------------|:--------------------------------------------|
 | `image.repository`           | controller container image repository                                                                     | quay.io/influxdb/chronograf                 |
```

Comment:
Is there a difference between “controller container image repository” and “image repository url”? Both are used as descriptions for the repository parameter in the Helm chart.

Reply:
`image repository url` is correct